### PR TITLE
feat: create card component

### PR DESCRIPTION
**Pull request by exu3**
create project card component with dummy data

closes Unfinished Card Component #15

### CODE REVIEW

**Comment by zaida04** [comment]
Gonna wait on Sarthaks review first since I'm not a frontend person.

**Comment by sarthaktexas** [comment]
Card component looks awesome but looks like the grid is messed up, let me check your code and i'll comment on it.

File: components/card.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-            <img
-                src="https://og-cards.vercel.app/Lorem%20Ipsum.png?theme=light&md=1&fontSize=225px&caption=A%2520website."
-                className="rounded-xl h-40 w-52"
-            />
-            <div className="flex flex-row">
-                <div className="flex flex-col ">
-                    <h2 className="font-rubik">Lorem ipsum</h2>
+        <div className="py-4">
+            <div
+                style={{ borderColor: `${featureColor}` }}
```
Comment (sarthaktexas) [inline components/card.js]: no need for a template string, remove it cause the variable can get passed it directly

File: pages/index.js [hunk 2]
```diff
@@ -15,18 +16,44 @@ export default function Home() {
...
                     <h2 className="font-rubik font-medium text-3xl md:w-2/3 lg:w-2/5">
-                        We're building a community for discovery and learning.
+                        We&#39;re building a community for discovery and learning.
                     </h2>
                     <div className="font-rubik font-medium bg-black text-center rounded-full px-3 py-1 text-lg text-white w-48">
                         Learn more. -&gt;
                     </div>
                 </section>
-                <svg className="transform rotate-180 -mt-1 md:-mt-5" viewBox="0 0 1440 320">
+                <svg className="-z-5 transform rotate-180 -mt-1 md:-mt-5" viewBox="0 0 1440 320">
```
Comment (sarthaktexas) [inline pages/index.js]: what's this class? that's not a thing in tailwind iirc

File: pages/index.js [hunk 3]
```diff
@@ -15,18 +16,44 @@ export default function Home() {
...
                     />
                 </svg>
+                <div className="z-40 justify-center flex flex-wrap space-x-12">
+                    <Card
+                        name="Student Name"
+                        title="Lorem Ipsum Dddddddd"
+                        projectType="Website"
+                        thumbnail="https://source.unsplash.com/random"
+                        profileIcon="https://picsum.photos/200"
+                        featureColor="gold"
```
Comment (sarthaktexas) [inline pages/index.js]: change the prop to award maybe??

File: pages/index.js [hunk 4]
```diff
@@ -15,18 +16,44 @@ export default function Home() {
...
+                    />
+                    <Card
+                        name="Student Name"
+                        title="Lorem Ipsum"
+                        projectType="Discord Bot"
```
Comment (sarthaktexas) [inline pages/index.js:43]: why are we using z indices everywhere

File: pages/index.js [hunk 5]
```diff
@@ -15,18 +16,44 @@ export default function Home() {
...
+                        profileIcon="https://picsum.photos/200"
+                        featureColor="silver"
+                    />
+                    <Card
+                        name="Student Name"
+                        title="Lorem Ipsum"
+                        projectType="Website"
+                        thumbnail="https://source.unsplash.com/random"
+                        profileIcon="https://picsum.photos/200"
+                        featureColor="white"
```
Comment (sarthaktexas) [inline pages/index.js]: there's no white feature color?

File: components/card.js [hunk 6]
```diff
@@ -1,21 +1,21 @@
...
+                className="rounded-xl h-40 w-72 overflow-hidden border-4">
```
Comment (sarthaktexas) [inline components/card.js]: don't overflow hidden, that doesn't make sense here

File: pages/index.js [hunk 7]
```diff
@@ -15,18 +16,44 @@ export default function Home() {
...
                     </div>
                 </section>
-                <svg className="transform rotate-180 -mt-1 md:-mt-5" viewBox="0 0 1440 320">
+                <svg className="-z-5 transform rotate-180 -mt-1 md:-mt-5" viewBox="0 0 1440 320">
                     <path
                         fill="#c1e8ed"
                         d="M0 64h24c24 0 72 0 120 5.3 48 5.7 96 15.7 144 37.4 48 21.3 96 53.3 144 80 48 26.3 96 48.3 144 64C624 267 672 277 720 272s96-27 144-42.7c48-16.3 96-26.3 144-32 48-5.3 96-5.3 144 16 48 21.7 96 63.7 144 69.4 48 5.3 96-26.7 120-42.7l24-16v96H0z"
                     />
                 </svg>
+                <div className="z-40 justify-center flex flex-wrap space-x-12">
```
Comment (sarthaktexas) [inline pages/index.js]: this is the problem, use grid instead of flexbox, flexbox is used for layouts that follow row and column style

**Review by sarthaktexas — CHANGES_REQUESTED** [review]
fix all the comments, dm me on slack or discord if you don't understand why i said it or if you need help fixing it

File: components/card.js (hunk 6 above)
Comment (exu3) [reply components/card.js]: without overflow hidden, the image flows outside the dimensions of the div. is there a better solution to this than `overflow-hidden`?
image overflowing
)

**Review by sarthaktexas — APPROVED** [review]
great

**Comment by sarthaktexas** [comment]
can somebody else mark this as resolved

**Review by zaida04 — APPROVED** [review]

**Pull request merged**